.github/workflows/ci.yml

```diff
@@ -50,7 +50,7 @@ jobs:
     strategy:
       matrix:
         # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
-        node-version: ["20.x", "22.x"]
+        node-version: ["22.x"]
```

Comment:
I don't think there's much point keeping the build against Node 20 any more, but I think keeping the matrix will allow us to add Node 24 when it becomes available.